model/Core/Classes/NamespaceMap.md

```diff
@@ -14,40 +14,24 @@ of ElementIDs to be used in SPDX content serialization in order to provide a mor
+1) An SPDX content producer wishing to provide clarity of their serialization of an SPDX 2.X simple style collection where all content is newly minted and a single prefix-namespace is used.  The consumer of SPDX content wishes to preserve the name space mapping provided by such a producer.  In this case, the consumer would record the namespace map prefixes in the NamespaceMap such that subsequent serializations could reproduce the prefixes / namespaces in the native serialization format.
+2) An SPDX content producer wishing to maintain consistent prefix use and understanding across multiple different serialization formats of the produced content.
+   For example, an SBOM producer wishes to share/publish the SBOM as JSON-LD and XML. The producer can specify the preferred prefix mappings in the native serialization format using the information from a single Namespace stored in memory.
 3) An SPDX content consumer/producer wishing to maintain consistent prefix use while round tripping from SPDX content received, deserialized, modified/extended in some way, and then reserialized in the same serialization form.
-   In this case the prefix-namespace mappings utilized in the content are preserved across serializations as native SPDX content. 
-4) An SPDX content consumer/producer wishing to maintain consistent prefix use while trans-serializing SPDX content received in one serialization form, deserialized, and then reserialized into a different serialization form.
-   In this case the prefix-namespace mappings utilized in the content are preserved across serializations as native SPDX content. 
-5) An SPDX content producer wishing to serialize/share together multiple collections of SPDX content each with their own namespace map in a way any consumer of the content would be able to separate out the multiple collections including their namespace map suggestions.
-   This would involve the conflicted mappings issue briefly characterized above this list of use cases.
-6) An SPDX content consumer wishing to maintain consistent prefix use while receiving serialized content that does not include a namespace map but does utilize prefixes, and at some future point reserializing that content.
-   The consumer can simply "wrap" the received content in a collection with a namespace map and specify the prefix to namespace mappings that were actually implemented in the received content.
-7) It should be possible to derive and maintain namespace mapping provenance for content.
+   In this case the prefix-namespace mappings utilized in the content are transformed from the original native namespace/prefix into the in memory NamespaceMap then transformed from the NamespaceMap back into the resultant serialization native namespace / prefix format.
```

Comment:
If the namespace map that the consumer is constructing from received content is only in memory (ephemeral) and not even persisted internally then this proposed consumer-centric approach will only ever offer consistency for a single "hop". This question of what this proposal proposes NamespaceMap to be is still quite unclear to me.

Reply:
Clarified by the correction above - locally, not in memory